src/app/components/views/job-view/subviews/customer-subview/customer-subview.component.html

```diff
@@ -0,0 +1,59 @@
+<div class="container mt-3 pb-2 pt-1 white-bg">
+  <ng-container *ngIf="job && job.customer">
+    <div class="row">
+      <h1>{{ job.customer.name }}</h1>
+    </div>
+    <div class="row subheading">
+      <span>CUSTOMER</span>
+    </div>
+    <div class="row">
+      <div class="col">
+        <span>{{ job.customer.name }}</span>
```

Comment:
elvis operators.  and all accessing below

Reply:
done